meta/main.yml

```diff
@@ -0,0 +1,22 @@
+galaxy_info:
```

Comment:
Nope

Reply:
Copy the file from ansible_skeleton and populate it.